src/atoms/Text/Text.stories.js

```diff
@@ -19,4 +19,16 @@ storiesOf('Text', module)
     <Text size={select('Size', sizeOptions, 'large')}>
       {text('Display Text', 'The quick brown fox jumps over the lazy dog')}
     </Text>
+  ))
+  .add('as span', () => (
+    <Text
+      size={select('Size', sizeOptions, 'large')}
+      as="span"
+      maxLines={select('maxLines', 'number of lines', 3)}
```

Comment:
This doesn't work. you need to provide the list of options
`select('maxLines', maxLinesOptions, 3)`
The as prop should also become the knob with few most possible values used `div, span, p,`

Reply:
done